drizzlepac/alignimages.py

```diff
@@ -700,174 +818,188 @@ def determine_fit_quality(imglist,filteredTable, print_fit_parameters=True):
+        dict_key = "{},{}".format(image_name, chip_num)
+        fit_status_dict[dict_key] = {'valid': False,
                                   'max_rms': max_rms_val,
                                   'num_matches': num_xmatches,
                                   'compromised': False,
-                                  'reason': ""} # Initialize dictionary entry for current image/chip
-        #Handle fitting failures (no matches found)
-        if item.meta['fit_info']['status'].startswith("FAILED") == True:
-                log.warning("No cross matches found in any catalog for {} - no processing done.".format(image_name))
-                continue
+                                  'reason': ""}
+        # Handle fitting failures (no matches found)
+        if item.meta['fit_info']['status'].startswith("FAILED") is True:
+            log.warning("No cross matches found in any catalog for {} "
+                        "- no processing done.".format(image_name))
+            continue
         fit_rms_val = item.meta['fit_info']['FIT_RMS']
         max_rms_val = item.meta['fit_info']['TOTAL_RMS']
         num_xmatches = item.meta['fit_info']['nmatches']
-        fitStatusDict[dictKey]['max_rms'] = max_rms_val
-        fitStatusDict[dictKey]['num_matches'] = num_xmatches
+        fit_status_dict[dict_key]['max_rms'] = max_rms_val
+        fit_status_dict[dict_key]['num_matches'] = num_xmatches
 
         if num_xmatches < MIN_CROSS_MATCHES:
-            if catalogIndex < numCatalogs-1:
-                log.warning("Not enough cross matches found between astrometric catalog and sources found in {}".format(image_name))
+            if catalogs_remaining:
+                log.warning(
+                    "Not enough cross matches found between astrometric"
+                    "catalog and sources found in {}".format(image_name))
                 continue
 
         # Execute checks
-        nmatchesCheck = False
+        nmatches_check = False
         if num_xmatches > 4:
-            nmatchesCheck = True
+            nmatches_check = True
 
-        radialOffsetCheck = False
-        radialOffset = math.sqrt(float(item.meta['fit_info']['shift'][0])**2 +
-                                 float(item.meta['fit_info']['shift'][0])**2)*item.wcs.pscale #radial offset in arssec
-        if float(num_xmatches) * 0.36 > 0.8 + (radialOffset/10.0)**8:
-            radialOffsetCheck = True
+        radial_offset_check = False
+        radial_offset = math.sqrt(
+            float(item.meta['fit_info']['shift'][0])**2 +
+            float(item.meta['fit_info']['shift'][0])**2) * item.wcs.pscale  # radial offset in arssec
+        if float(num_xmatches) * 0.36 > 0.8 + (radial_offset / 10.0)**8:
+            radial_offset_check = True
 
-        largeRmsCheck = True
+        large_rms_check = True
         if fit_rms_val > 150. or max_rms_val > 150.:
-            largeRmsCheck = False
+            large_rms_check = False
```

Comment:
@jhunkeler Please, can we do this: `large_rms_check = fit_rms_val > 150. or max_rms_val > 150.`